src/content/en/fundamentals/performance/rail.md

```diff
@@ -9,15 +9,16 @@ description: RAIL is a user-centric performance model. Every web app has these f
 
 {% include "web/_shared/contributors/megginkearney.html" %}
 
-RAIL is a user-centric performance model. Every web app has these four distinct aspects to its life cycle, and performance fits into them in very different ways:
+RAIL is a user-centric performance model. Every web app has these four distinct aspects to its life cycle, and performance fits into them in different ways:
 
 ![RAIL performance model](images/rail.png)
 
 
 ### TL;DR {: .hide-from-toc }
-- Focus on the user; the end goal isn't to make your site perform fast on any specific device, it''s to ultimately make users happy.
+
+- Focus on the user; the end goal isn't to make your site perform fast on any specific device, it's to ultimately make users happy.
 - Respond to users immediately; acknowledge user input in under 100ms.
-- Render each frame in under 16ms and aim for consistency; users notice 'jank'.
+- Execute animation code in under 10ms.
 - Maximize main thread idle time.
```

Comment:
IIRC we've been telling developers to render a frame in 16ms, but to complete specific pieces of work in far less time than that (10ms). Is there a way for us to still highlight frame budget of 16.6ms but also say execution time for animation code should be 10ms?.